packages/widget/src/utils/index.ts

```diff
@@ -84,3 +84,16 @@ export const validateAddress = (
       return 'unsupported network';
   }
 };
+
+export const debounceAmountChange = <Args extends string>(
+  cb: (args: Args) => void,
+  delay?: number
```

Comment:
why is `delay` optional? if it should remain so, it'd be good to put a default value

Reply:
change it to required